dtls.c

```diff
@@ -714,10 +715,30 @@ static const char *dtls_handshake_type_to_name(int type)
   case DTLS_HT_FINISHED:
     return "finished";
   default:
-    return "unknown";
+    snprintf(buf, sizeof(buf), "unknown (%d)", type);
+    return buf;
   }
 }
 
+static const char *dtls_message_type_to_name(int type)
+{
+ static char buf[16];
+  switch (type) {
+  case DTLS_CT_CHANGE_CIPHER_SPEC:
+    return "change_cipher_spec";
+  case DTLS_CT_ALERT:
+    return "alert";
+  case DTLS_CT_HANDSHAKE:
+    return "handshake";
+  case DTLS_CT_APPLICATION_DATA:
+    return "application_data";
+  default:
+    snprintf(buf, sizeof(buf), "unknown (%d)", type);
```

Comment:
See previous comment.

Reply:
Updated as per previous comment and code pushed.